### ISSUE TITLE
compilation failure: error C2440: 'initializing': cannot convert from 'nullptr' to 'VkBuffer'

### ISSUE THREAD

**Issue by drichardson** [godotengine/godot#36113]
**Godot version:**
e9d9fef7208b21fc963e9064e8d3337ab8769ca2

**OS/device including version:**
- Windows 10 Version 10.0.18363 Build 18363
- Visual Studio 2019 16.4.3
- scons v3.1.2.bee7caf9defd6e108fc2998a2520ddb36a967691
- cl 19.24.28315 for x86

**Issue description:**
Build fails.

```
[ 57%] Compiling ==> thirdparty\libpng\pngmem.c
pngmem.c
E:\code\godot\thirdparty\vulkan\vk_mem_alloc.h(14148): error C2440: 'initializing': cannot convert from 'nullptr' to 'VkBuffer'
E:\code\godot\thirdparty\vulkan\vk_mem_alloc.h(14148): note: A native nullptr can only be converted to bool or, using reinterpret_cast, to an integral type
scons: *** [thirdparty\vulkan\vk_mem_alloc.windows.tools.32.obj] Error 2
scons: building terminated because of errors.
```

**Steps to reproduce:**

1. Open Developer Command Prompt for VS 2019 (v16.4.3) (this is an x86 environment)
1. `scons -j6 platform=windows -c`
1. `scons -j6 platform=windows`

**Minimal reproduction project:**
n/a

**Comment by bruvzg** [comment]
Should be fixed by updating VulkanMemoryAllocator (see #36107).

**Comment by drichardson** [comment]
Yup. Closing this issue as a dupe.
